ui/raidboss/data/04-sb/dungeon/bardams_mettle.txt

```diff
@@ -0,0 +1,169 @@
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839:.*is no longer sealed/ window 5000 jump 0
+
+#~~~~~~~~#
+# Garula #
+#~~~~~~~~#
+
+# -ic "Steppe Coeurl" "Steppe Sheep" "Steppe Yamaa"
+
+0 "Start" sync / 00:0839:Bardam's Hunt will be sealed off/
+11.7 "Heave" sync /:Garula:1EF7:/ window 11.7,5
+20.9 "Crumbling Crust (cast)" sync /:Garula:1EF8:/
+22.4 "Crumbling Crust (circles)" sync /:Garula:1F13:/
+34.1 "Rush" sync /:Garula:1EF9:/
+36.2 "War Cry" sync /:Garula:1EFA:/
+38.9 "Earthquake" sync /:Garula:1EFB:/ window 15,15
+49.5 "Heave" sync /:Garula:1EF7:/
+58.7 "Crumbling Crust (cast)" sync /:Garula:1EF8:/
+60.2 "Crumbling Crust (circles)" sync /:Garula:1F13:/
+
+66.6 "Heave" sync /:Garula:1EF7:/ jump 11.7
```

Comment:
I never knew this was such a short loop!

Reply:
Everything in here felt so much longer before I built the timeline. The Yol fight feels really intense sometimes, but it's only 8-10 total mechanics!